README.md

```diff
@@ -223,8 +223,7 @@ This header also defines `__rtsan::ScopedDisabler()`, which allows for disabling
 - [x] Ability to disable error reporting in a specified scope (`__rtsan::ScopedDisabler`)
 - [x] Raise an error if a function attributed with `[[clang::blocking]]` is called in a real-time context
 - [x] `Continue` error mode - using `halt_on_error` runtime flag
-- [ ] Suppressing specific errors
-    - In progress [here](https://github.com/llvm/llvm-project/pull/111608)
+- [x] Suppressing specific errors
```

Comment:
Do we want to remove this entire section now that we have done all our v1 things?

I feel like having no unchecked boxes implies "finished", and while we have more stuff to do (more interceptors, little tweaks) it is nothing as substantial as the above.

Reply:
Yes, good idea. I'll push up a hotfix now if you like?